include/envoy/network/connection.h

```diff
@@ -79,24 +79,24 @@ class DynamicSocketOptions {
   /**
    * Set the receive buffer size for the socket to a value specified by the 'buffer_size'
    */
-  virtual void setSocketRecvBufferSize(uint32_t buffer_size) const PURE;
+  virtual Api::SysCallIntResult setSocketRecvBufferSize(int buffer_size) const PURE;
```

Comment:
Also sorry I missed this last time but please comment include/* functions with @param and @return

Reply:
Got it. I will make this change.